docker/gcloud_build.sh

```diff
@@ -2,8 +2,8 @@
 
 set -ex
 
-gcloud config set project istio-test
+gcloud config set project istio-testing
```

Comment:
gcloud docker --authorize-only
and then you can push to any gcr bucket you have permissions for.